### ISSUE TITLE
AdoptOpenJDK 16.0.1 compatibility

### ISSUE THREAD

**Issue by dig** [influxdata/influxdb-client-java#260]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. Install AdoptOpenJDK 16.0.1
2. Try to connect to any influxdb server
3. Error below occurs
```
java.lang.ExceptionInInitializerError: null
	at java.lang.J9VMInternals.ensureError(J9VMInternals.java:184) ~[?:?]
	at java.lang.J9VMInternals.recordInitializationFailure(J9VMInternals.java:173) ~[?:?]
	at retrofit2.Retrofit$Builder.<init>(Retrofit.java:441) ~[?:?]
	at com.influxdb.client.internal.AbstractInfluxDBClient.<init>(AbstractInfluxDBClient.java:95) ~[?:?]
	at com.influxdb.client.internal.InfluxDBClientImpl.<init>(InfluxDBClientImpl.java:97) ~[?:?]
	at com.influxdb.client.InfluxDBClientFactory.create(InfluxDBClientFactory.java:204) ~[?:?]
	at com.influxdb.client.InfluxDBClientFactory.create(InfluxDBClientFactory.java:159) ~[?:?]
	at net.sunken.velocity.VelocityProxyPlugin.onProxyInitialization(VelocityProxyPlugin.java:53) ~[?:?]
	at net.sunken.velocity.Lmbda$7.execute(Unknown Source) ~[?:?]
	at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:47) ~[velocity-3.0.x-SNAPSHOT.jar:3.0.1-SNAPSHOT (git-80875c0b-bunknown)]
	at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler$$Lambda$376/0x00000000a5118538.executeAsync(Unknown Source) ~[?:?]
	at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:585) ~[velocity-3.0.x-SNAPSHOT.jar:3.0.1-SNAPSHOT (git-80875c0b-bunknown)]
	at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$5(VelocityEventManager.java:466) ~[velocity-3.0.x-SNAPSHOT.jar:3.0.1-SNAPSHOT (git-80875c0b-bunknown)]
	at com.velocitypowered.proxy.event.VelocityEventManager$$Lambda$378/0x00000000a5119f70.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[?:?]
	at java.lang.Thread.run(Thread.java:853) [?:?]
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int) accessible: module java.base does not "opens java.lang.invoke" to unnamed module @56a7fc28
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:357) ~[?:?]
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297) ~[?:?]
	at java.lang.reflect.Constructor.checkCanSetAccessible(Constructor.java:188) ~[?:?]
	at java.lang.reflect.Constructor.setAccessible(Constructor.java:181) ~[?:?]
	at retrofit2.Platform.<init>(Platform.java:59) ~[?:?]
	at retrofit2.Platform.findPlatform(Platform.java:44) ~[?:?]
	at retrofit2.Platform.<clinit>(Platform.java:35) ~[?:?]
	... 19 more
```

__Expected behavior:__
Describe what you expected to happen.

Allow connection to influxdb on java 16.

__Actual behavior:__
Describe What actually happened.

Error while connecting to influxdb while using java 16.

__Specifications:__
 - Client Version: `influxdb-client-java:3.2.0`
 - InfluxDB Version: `InfluxDB OSS 2.0`
 - JDK Version: `OpenJDK Runtime Environment AdoptOpenJDK-16.0.1+9 (build 16.0.1+9`
 - Platform: `Ubuntu 18.04`

**Comment by bednar** [comment]
Hi @dig,

thanks for using our client.

It looks like a problem with `Retrofit` library and AdoptOpenJDK. It will be fixed by https://github.com/square/retrofit/pull/3618.

The our client is tested with OpenJDK, are you able to switch from AdoptOpenJDK to OpenJDK?

Regards

**Comment by dig** [comment]
> Hi @dig,
> 
> thanks for using our client.
> 
> It looks like a problem with `Retrofit` library and AdoptOpenJDK. It will be fixed by [square/retrofit#3618](https://github.com/square/retrofit/pull/3618).
> 
> The our client is tested with OpenJDK, are you able to switch from AdoptOpenJDK to OpenJDK?
> 
> Regards

Hi,

I am currently unable to switch to OpenJDK, do you know when this will be fixed?

Thank you.

**Comment by bednar** [comment]
> I am currently unable to switch to OpenJDK, do you know when this will be fixed?

I don't know, but you can add following options to your JVM as a workaround: `--add-opens=java.base/java.lang.invoke=ALL-UNNAMED`.
